src/tapis-app/src/Sections/Apps/Apps.tsx

```diff
@@ -24,11 +26,17 @@ const Apps: React.FC = () => {
       <ListSectionHeader>Apps</ListSectionHeader>
       <ListSectionBody>
         <ListSectionList>
-          <AppsListing />
+          <AppsListing onSelect={appSelectCallback} />
         </ListSectionList>
         <ListSectionDetail>
+          <ListSectionHeader type={"sub-header"}>Launcher</ListSectionHeader>
           <div>
-            {selectedApp ? selectedApp.id : "Select an app from the list"}
+            {selectedApp
+              ? <Launcher />
```

Comment:
ok i think initialValues prop should be ported over from the old Launcher component in tapis-app:

```
  const appId = selectedApp.id;
  const appVersion: selectedApp.version;
  const initialValues: Jobs.ReqSubmitJob = {
    appId,
    appVersion,
    name: `${appId}-${appVersion}-${new Date().toISOString().slice(0, -5)}`,
    execSystemId: 'tapisv3-exec'
  }
```